gap/oper.gi

```diff
@@ -1725,6 +1725,36 @@ function(D, list)
   return DigraphShortestDistance(D, list[1], list[2]);
 end);
 
+InstallMethod(IsReachableFrom, "for a digraph and a vertex",
```

Comment:
I'd suggest a different name. In GAP, I expect `Is...` functions to be things that return either `true` or `false`. Something more appropriate could be:
* `ReachableFrom`
* `VerticesReachableFrom`

Reply:
Thanks @wilfwilson! Did that now! :)